.github/workflows/fast-dispatch-full-regressions-and-models-impl.yaml

```diff
@@ -40,6 +40,13 @@ jobs:
               cmd: tests/scripts/single_card/nightly/run_ttnn.sh,
               timeout: 70
             },
+            {
+              name: "WH N300 pgm dispatch nightly",
+              arch: wormhole_b0,
+              runs-on: ["cloud-virtual-machine", "N300", "in-service"],
+              cmd: ./tests/tt_metal/tt_metal/perf_microbenchmark/dispatch/compare_pgm_dispatch_perf_ci.sh,
+              timeout: 70
```

Comment:
Can we reduce this timeout?

Reply:
This is in minutes right? I think 10 could work?

Reply:
Yes